functionnal_tests/cypress/integration/app_file/file_name_spec.js

```diff
@@ -0,0 +1,49 @@
+import { PAGES } from '../../support/urls_commands'
+
+const threadTitle = 'Thread'
+const fileTitle = 'File'
+const fullFilename = 'Linux-Free-PNG.png'
+const contentType = 'image/png'
+let threadId
+let fileId
+let workspaceId
+
+describe("An app's name", () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+      cy.createFile(fullFilename, contentType, fileTitle, workspaceId).then(file => fileId = file.content_id)
+      cy.createThread(threadTitle, workspaceId).then(thread => threadId = thread.content_id)
+    })
+  })
+
+  beforeEach(function () {
+    cy.loginAs('administrators')
+  })
+
+  afterEach(() => {
+    cy.cancelXHR()
+  })
+
+  it('should show the file extension with content type is a file', () => {
+    cy.visitPage({
+      pageName: PAGES.CONTENT_OPEN,
+      params: { workspaceId: workspaceId, contentType: 'file', contentId: fileId }
+    })
+    cy.get('.newVersionBtn').click()
+    cy.dropFixtureInDropZone(fullFilename, contentType, '.filecontent__form', fullFilename)
+    cy.get('.file__contentpage__dropzone__btn__validate').click()
+    cy.contains('.FilenameWithExtension .badge', '.png')
+  })
+
+  it('should not show the file extension with content type is not a file', () => {
```

Comment:
I'd like to point out that this test doesn't seem related to the app file since it opens the app thread.
You are actually testing the component FilenameWithExtension in this file if I'm right.
Since this file is in frontend_lib, I don't know where to put it though. So, no changes required

Reply:
Maybe put at `app_interface`?

Reply:
that would go to a new folder maybe called "apps" to me if we want to handle this, but I'm fine with keeping it in app_file or putting it in app_interface

Reply:
There is a `generic_frontend` folder in integration.

Reply:
indeed, but I think this folder is related to components for the frontend/ folder, not the frontend_lib/ one.

I'm fine with app_interface too